sbncode/PID/scripts/ShowerPIDMVA.C

```diff
@@ -0,0 +1,65 @@
+#include "TChain.h"
+#include "TFile.h"
+#include "TObjString.h"
+#include "TROOT.h"
+#include "TString.h"
+#include "TSystem.h"
+#include "TTree.h"
+
+#include "TMVA/DataLoader.h"
+#include "TMVA/Factory.h"
+#include "TMVA/TMVAGui.h"
+#include "TMVA/Tools.h"
+
+void ShowerPIDMVA(TString fileName)
+{
+    gStyle->SetOptStat(0);
+    TFile f(fileName);
+
+    TFile *outputFile = TFile::Open("ShowerPIDMVA.root", "RECREATE");
+
+    TMVA::Factory *factory = new TMVA::Factory("ShowerPIDMVA", outputFile, "!V:!Silent:Color:DrawProgressBar:AnalysisType=multiclass");
+    TMVA::DataLoader *dataloader = new TMVA::DataLoader("dataset");
+
+    TDirectory *dir   = f.GetDirectory("pandoraShowerSBNMVAPID");
+    TTree *showerTree = (TTree *)dir->Get("showerTree");
+
+    TTree *electronTree = showerTree->CopyTree("std::abs(truePdg)==11");
+    TTree *photonTree   = showerTree->CopyTree("std::abs(truePdg)==22");
+    TTree *otherTree    = showerTree->CopyTree("std::abs(truePdg)!=11 && std::abs(truePdg)!=22");
+
+    dataloader->AddTree(electronTree, "Electron");
+    dataloader->AddTree(photonTree, "Photon");
+    dataloader->AddTree(otherTree, "Other");
+
+    dataloader->AddVariable("bestdEdx", "Best Plane dEdx", "MeV/cm", 'F', 0, 10);
+    dataloader->AddVariable("convGap", "Conversion Gap", "cm", 'F', 0, 10);
+    dataloader->AddVariable("openAngle", "Opening Angle", "rad", 'F', 0, 10);
+    dataloader->AddVariable("modHitDensity", "Modified Hit Density", "", 'F', 0, 10);
+    dataloader->AddVariable("sqrtEnergyDensity", "Sqrt Energy Density", "", 'F', 0, 10);
+    // dataloader->AddVariable("logEnergyDensity", "Sqrt Energy Density", "cm^-1", 'F', 0, 10);
+    // 0, 10); dataloader->AddVariable("", "", "", 'F', 0, 10); dataloader->AddVariable("", "", "", 'F', 0, 10);
```

Comment:
I can understand keeping `logEnergyDensity` for future use but what about this line?

Reply:
Fixed